src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -85,6 +88,11 @@
   // This needs to be volatile as the RcaConfPoller writes it but the Nanny reads it.
   private static volatile boolean rcaEnabled = false;
 
+  private String mutedRcasDefaultValue = "";
+
+  // This needs to be volatile as the RcaConfPoller writes it but the Nanny reads it.
+  private static volatile String mutedRcas = "";
```

Comment:
I am thinking instead of storing a list of rcas in a comma separated string, we should probably process it and it should be an array of Strings ?

Reply:
Agreed. 
Initially I had planned to use an array of strings for mutedRCA cluster settings but ES cluster settings only support list type with special handling. Since we are no longer relying on cluster settings, I will revert this to use an array of strings.